index.js

```diff
@@ -29,10 +39,17 @@ export default class Flatbush {
         if (!ArrayType) {
             throw new Error('Unrecognized array type.');
         }
-        const [nodeSize] = new Uint16Array(data, 2, 1);
-        const [numItems] = new Uint32Array(data, 4, 1);
-
-        return new Flatbush(numItems, nodeSize, ArrayType, undefined, data);
+        const [nodeSize] = new Uint16Array(data, byteOffset + 2, 1);
+        const [numItems] = new Uint32Array(data, byteOffset + 4, 1);
+
+        return new Flatbush(
+            numItems,
+            nodeSize,
+            ArrayType,
+            undefined,
+            data,
+            byteOffset
+        );
```

Comment:
Nit: let's keep the original formatting where possible. I like things more compact.